xbmc/cores/VideoPlayer/VideoRenderers/RenderCapture.cpp

```diff
@@ -127,13 +127,26 @@ void CRenderCaptureGL::BeginRender()
   if (!m_asyncChecked)
   {
 #ifndef HAS_GLES
-    m_asyncSupported = CServiceBroker::GetRenderSystem()->IsExtSupported("GL_ARB_pixel_buffer_object");
-    m_occlusionQuerySupported = CServiceBroker::GetRenderSystem()->IsExtSupported("GL_ARB_occlusion_query");
-
+    unsigned int major, minor, glversion;
+    CServiceBroker::GetRenderSystem()->GetRenderVersion(major, minor);
+    glversion = 10 * major + minor;
+    if (glversion >= 21)
+    {
+      m_asyncSupported = true;
+      m_occlusionQuerySupported = true;
+    }
+    else if (glversion > 14)
+    {
+      m_occlusionQuerySupported = true;
+    }
+    else
+    {
+      CLog::Log(LOGWARNING, "CRenderCaptureGL: GL_occlusion_query not supported, upgrade your GL drivers to support at least GL 2.1");
+    }
     if (m_flags & CAPTUREFLAG_CONTINUOUS)
     {
       if (!m_occlusionQuerySupported)
-        CLog::Log(LOGWARNING, "CRenderCaptureGL: GL_ARB_occlusion_query not supported, performance might suffer");
+        CLog::Log(LOGWARNING, "CRenderCaptureGL: GL_occlusion_query not supported, performance might suffer");
```

Comment:
```suggestion
        CLog::Log(LOGWARNING, "CRenderCaptureGL: Occlusion queries not supported, performance might suffer");
```

There is no `GL_occlusion_query` :-)

Reply:
Jep - thx.